File: Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        listmap = {}
        
        for i, value in enumerate(nums):
            diff = target - value
                
            if diff in listmap:
                return(listmap[diff], i)
            listmap[value] = i
        return

File: Two_Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector <int> ans(2);
        for(int i=0 ; i <  nums.size(); i++) {
            int diff = target - nums[i];
            for(int j=i+1;  j< nums.size();j++){
                if(diff == nums[j]){
                    ans[0] = i;
                    ans[1] = j;
                
                }    
            }
        }
        return ans;
    }
   
};
